polygon/rest/models/snapshot.py

```diff
@@ -214,3 +214,145 @@ def from_dict(d):
             spread=d.get("spread", None),
             updated=d.get("updated", None),
         )
+
+
+@modelclass
+class UniversalSnapshotSession:
+    """Contains data about the most recent trading session for an asset."""
+
+    price: Optional[float] = None
+    change: Optional[float] = None
+    change_percent: Optional[float] = None
+    early_trading_change: Optional[float] = None
+    early_trading_change_percent: Optional[float] = None
+    late_trading_change: Optional[float] = None
+    late_trading_change_percent: Optional[float] = None
+    open: Optional[float] = None
+    close: Optional[float] = None
+    high: Optional[float] = None
+    low: Optional[float] = None
+    previous_close: Optional[float] = None
+    volume: Optional[float] = None
+
+    @staticmethod
+    def from_dict(d):
+        return UniversalSnapshotSession(**d)
```

Comment:
the previous model has `vwap` instead of `price`, and has `last_updated` which the new one does not